packages/discovery-provider/plugins/pedalboard/apps/relay/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "tsc",
     "build:watch": "tsc --watch",
     "clean": "rm -rf dist",
-    "dev": "ts-node-dev --respawn ./src/index.ts",
+    "dev": "nodemon ./dist/index.js",
```

Comment:
pardon my snooping, saw the pr and was curious! was ts-node-dev causing issues?

Reply:
this was causing relay not to build within the container, couldn't link to basekit for some reason

Reply:
i think there's a way for ts-node-dev to work but since turbo is handling the build thought i'd just put nodemon on the transpiled js

Reply:
this won't be an issue once I move all the plugins up into packages as well